### ISSUE TITLE
Collaboration

### ISSUE THREAD

**Issue by the-ethan-hunt** [the-ethan-hunt/first-timers-guide#23]
We need collaboration with societies and people to promote this and open-source. issue open for discussions

**Comment by ap1459** [comment]
Hi @the-ethan-hunt, what kind of societies do you have in mind? I can also promote this by sharing it with other students and asking them for contributions if that's what you mean?

**Comment by the-ethan-hunt** [comment]
@ap1459 , yes in a way. First-timers-guide is currently collaborating with CMoC. This guide is for those who are new to open-source. Hence I desire to promote it through various coding programmes in various institutes. If your college has a 'code-club', you can certainly tell them about this; but not ask for contributions. :smile:
